app/vacancy_sources/vacancy_source/source/united_learning.rb

```diff
@@ -50,10 +50,6 @@ def each
 
   private
 
-  def vacancy_listed_at_excluded_school_type?(school)
-    EXCLUDED_DETAILED_SCHOOL_TYPES.include?(school.detailed_school_type) if school
-  end
-
```

Comment:
From what I see, this only differed from the original method in the fact that operates over an individual school as input instead of a collection. 
Opted to use the original method, passing a collection with a single school as an argument.